include/status-code/status_code.hpp

```diff
@@ -134,6 +134,11 @@ namespace detail
     static constexpr bool value = true;
   };
 
+  template <class T> struct remove_cvref
```

Comment:
If `__cpp_lib_remove_cvref` then polyfill with C++ 20 `remove_cvref`.

(One day future compilers may compile faster if you use std traits directly)